recipes/fsl-pyfeeds/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "fsl-pyfeeds" %}
+{% set version = "0.8.0" %}
+{% set sha256 = "7c10d6b3ab242ca20b28bab2b0298260c5f5bf42893f4052491fc2e9ab6b8f73" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
this is not necessary

Reply:
Do you mean the `fn` key-value pair? I guess it's not. I think I have based this on an old version of `staged-recipes/recipes/examples/meta.yaml`..